CustomApps/lyrics-plus/index.js

```diff
@@ -323,13 +323,59 @@ class LyricsContainer extends react.Component {
 			if (CACHE[info.uri]?.[CONFIG.modes[mode]]) {
 				this.resetDelay();
 				this.setState({ ...CACHE[info.uri], isCached });
+				{
+					let mode = -1;
+					if (this.state.explicitMode !== -1) {
+						mode = this.state.explicitMode;
+					} else if (this.state.lockMode !== -1) {
+						mode = this.state.lockMode;
+					} else {
+						// Auto switch
+						if (this.state.karaoke) {
+							mode = KARAOKE;
+						} else if (this.state.synced) {
+							mode = SYNCED;
+						} else if (this.state.unsynced) {
+							mode = UNSYNCED;
+						} else if (this.state.genius) {
+							mode = GENIUS;
+						}
```

Comment:
why would it? Mode is already `-1` by default

Reply:
Fair!

Reply:
But then the following would produce undefined
```js
CONFIG.modes[mode]
```
and then

```js
CACHE[info.uri][undefined]
```

Reply:
yea meaning there's no lyrics

Reply:
> and then
> 
> ```js
> CACHE[info.uri][undefined]
> ```

At the end it will just be undefined and will fail the if check. Also this code is taken nearly as is from the render function, so if there is/was a problem, it should be/have been fixed there first.